regression/cbmc/unsigned___int128/main.c

```diff
@@ -1,3 +1,8 @@
+#if defined(_WIN32) && !defined(__CYGWIN__)
+void reduce()
```

Comment:
I would check for the (absence of the) gcc define -- I believe __int128 is a gcc extension; so this isn't about Windows.